.github/stale.yml

```diff
@@ -15,4 +15,7 @@ issues:
         - bug
     markComment: >
         This issue has been automatically marked as stale because it has not had
-        recent activity. It will be closed if no further activity occurs.
+        any activity in the last 2 years. If you feel that this is a valid issue,
```

Comment:
maybe "If you feel that this issue is important"?  Just thinking that an issue could be valid, but nobody cares about it.  I guess people would only comment if they care though, so also okay if you leave as is.